hkl/util.py

```diff
@@ -220,6 +221,11 @@ def __repr__(self):
 Lattice = namedtuple("LatticeTuple", "a b c alpha beta gamma")
 
 
+def new_lattice(a, b=None, c=None, alpha=90, beta=None, gamma=None):
```

Comment:
```suggestion
def new_lattice(a, b=None, c=None, alpha=90., beta=None, gamma=None):
```

Probably useful down the line to already make the angles `float` values here.